internal/federationdomain/clientregistry/clientregistry_test.go

```diff
@@ -312,6 +340,43 @@ func requireEqualsPinnipedCLI(t *testing.T, c *Client) {
 		  "token_endpoint_auth_method": "none",
 		  "request_uris": null,
 		  "request_object_signing_alg": "",
-		  "token_endpoint_auth_signing_alg": "RS256"
+		  "token_endpoint_auth_signing_alg": "RS256",
+          "IDTokenLifetimeConfiguration": 0
 		}`, string(marshaled))
 }
+
+func requireDynamicOIDCClient(
+	t *testing.T,
+	c *Client,
+	wantClientID string,
+	wantHashes []string,
+	wantGrantTypes fosite.Arguments,
+	wantScopes fosite.Arguments,
+	wantRedirectURIs []string,
+	wantIDTokenLifetimeConfiguration time.Duration,
+) {
+	require.Equal(t, wantClientID, c.GetID())
+
+	require.Len(t, c.GetRotatedHashes(), len(wantHashes))
+	for i := range wantHashes {
+		require.Equal(t, wantHashes[i], string(c.GetRotatedHashes()[i]))
+	}
+
+	require.Equal(t, wantRedirectURIs, c.GetRedirectURIs())
+	require.Equal(t, wantGrantTypes, c.GetGrantTypes())
+	require.Equal(t, wantScopes, c.GetScopes())
+	require.Equal(t, wantIDTokenLifetimeConfiguration, c.IDTokenLifetimeConfiguration)
```

Comment:
Use Getter for this as well?

Reply:
Will add getter and update all readers to use getter.